dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsJdbcTask.java

```diff
@@ -196,6 +208,19 @@ private String getSql(@Nonnull JdbcHandle handle) {
                 + "AND L.StartTime < CAST('%s' AS TIMESTAMP)\n",
             SQL_FORMAT.format(interval.getStart()), SQL_FORMAT.format(interval.getEndExclusive())));
 
+    logDateColumn.ifPresent(
```

Comment:
Then this would become if logDateColumn != null

Reply:
Done.